index.js

```diff
@@ -237,3 +241,34 @@ function arrObjKeys (obj, inspect) {
     }
     return xs;
 }
+
+// Returns the object's constructor name or null if it is a plain object
+// or doesn't have a prototype.
+function getTypeString(o) {
+    if (Object.prototype.toString(o) !== '[object Object]') return null;
```

Comment:
I think you want `.toString.call(o)`, otherwise this will always return `true`

Reply:
You're right, my bad. I'll fix this.